fixtures/explo/gasprojector-gx2.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Gasprojector GX2",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Gloryfire"],
+    "createDate": "2019-05-30",
+    "lastModifyDate": "2019-05-30"
+  },
+  "links": {
+    "manual": [
+      "https://www.explo.at/_Resources/Persistent/459c14f10f9fddad03748a108704cb345c719266/Gasbrenner%20GX2%202.0%20EN.pdf"
+    ],
+    "productPage": [
+      "https://www.explo.at/en/effektgeraete/gas/gx2-12v.html"
+    ],
+    "video": [
+      "https://youtu.be/nRJiaLzVk0Y",
+      "https://www.youtube.com/watch?v=ut26zDS3Z4c"
+    ]
+  },
+  "physical": {
+    "dimensions": [260, 390, 260],
+    "weight": 6.8,
+    "DMXconnector": "3-pin"
+  },
+  "availableChannels": {
+    "Ignition": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 252],
+          "type": "NoFunction",
+          "comment": "No ignition"
+        },
+        {
+          "dmxRange": [253, 255],
+          "type": "Effect",
+          "effectName": "Single ignition"
+        }
+      ]
+    },
+    "Valve Opening Time": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction",
+          "comment": "Saftety zone"
+        },
+        {
+          "dmxRange": [11, 13],
+          "type": "Time",
+          "timeStart": "20ms",
+          "timeEnd": "60ms",
```

Comment:
This is actually a large 20…3800ms capability, but I thought the descriptions might be helpful.